chronos/internal/asyncengine.nim

```diff
@@ -10,6 +10,92 @@
 
 {.push raises: [].}
 
+## This module implements the core asynchronos engine / dispatcher.
```

Comment:
Is asynchronos a custom name (because it contains chronos) or is it a typo and supposed to be asynchronous?

Reply:
typo - didn't see that wordplay though, now I like it ;)